### DISCUSSION TITLE
Server listings on joinmastodon.org

### DISCUSSION THREAD

**Discussion by luebken** [mastodon/mastodon#19581] [General]
Where is the list of servers curated on https://joinmastodon.org/servers? What would be the best way to recommend adding a new one e.g. https://fosstodon.org/?

**Comment by pergatore** [comment]
+1 Gaming section seems to have four servers but there are a lot more great ones out there.

**Comment by Gargron** [comment]
It's manually curated, send an e-mail to hello@joinmastodon.org. fosstodon.org is already on there though.

**Reply by LuckyPengtao** [reply]
I have encountered the same problem. May I ask if your site has already been added to the list?

**Comment by janedzumerko** [comment]
Hey, I created an instance about [web performance, web development, and related fields](https://webperf.social/about). I sent an email to hello@joinmastodon.org so it can be listed on https://joinmastodon.org/servers but didn't get any reply.

**Reply by GitInvarion** [reply]
cf https://joinmastodon.org/covenant

_Because our goal is giving new users a smooth and reliable start into the decentralized social network of Mastodon rather than simply promoting individual servers, we are generally not interested in promoting brand new and unproven servers, so the submission should not be your primary strategy of growing your community._
